source/_references/searching.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Searching images
+use: [references]
+---
+
+## Intro
+
+It's possible to list images and their metadata in your organization and also filter by their metadata.
+The filter options are pretty basic right now, but may be improved in the future.
+
+
+## Listing source images
+
+Listing source images allows searching and sorting of images.
+The images are by default sorted by their uploaded date desc (latest first).
```

Comment:
actually we should say: `are by default sorted by their created date desc (latest first).`
because the field is named `created`